main.py

```diff
@@ -48,7 +48,7 @@
     extended_project_list_path = os.path.join(file_ops.extended_project_list_path(), 'extended_project_list.csv')
     extended_project_list = final_result['extended_project_list']
     extended_project_list.to_csv(extended_project_list_path, index=False)
-
+    validation_enabled = True
```

Comment:
Is there a reason validation_enabled is being force to `True`? that should be set with the command line option `--validate`